0.13.0/CONTRIBUTING.html

```diff
@@ -0,0 +1,311 @@
+
+
+<!DOCTYPE html>
+<html class="writer-html5" lang="en" >
+<head>
+  <meta charset="utf-8" />
+  
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  
+  <title>Contributing &mdash; numba-dppy 0.12.0 documentation</title>
```

Comment:
`numba-dppy 0.12.0` - old version.

Reply:
https://github.com/IntelPython/numba-dppy/blob/9c8dabf929368db96c3a2abf42072178b6cd9634/docs/conf.py#L26

Reply:
@PokhodenkoSA Can I replace all lines with version 0.12 to 0.13 in this PR?

Reply:
Bad solution. It is better to fix root cause.